sensor_msgs/CMakeLists.txt

```diff
@@ -79,6 +79,10 @@ if(cpp_typesupport_target)
   ament_export_targets(export_${PROJECT_NAME})
 
   if(BUILD_TESTING)
+    find_package(ament_cmake_gtest REQUIRED)
+    find_package(ament_lint_auto REQUIRED)
+    ament_lint_auto_find_test_dependencies()
```

Comment:
I think only the `find_package(ament_cmake_gtest)` line should be moved. The linter tests can be run without the C++ typesupport target.